pkg/dockerfile/fast_generator.go

```diff
@@ -428,6 +428,11 @@ func (g *FastGenerator) installSrc(lines []string, weights []weights.Weight) ([]
 		lines = append(lines, "RUN "+strings.Join(linkCommands, " && "))
 	}
 
+	if filepath.Base(g.Config.Filename()) != "cog.yaml" {
+		// Absolute filename doesn't work anyway, so it's always relative
```

Comment:
Cog mistakenly uses `/` as project root if `-f /abs/path/to/cog.yaml` and fails on creating `/.cog` :shrug: